content/pairing.md

```diff
@@ -0,0 +1,248 @@
+     ]
+    }
+   ],
+   "source": [
+    "x = 2 + 2\n",
+    "print('x = ', x)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.3"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
+```
+
+</td>
+
+<td>
+
+````
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.6.0
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+This code calculates 2+2 and prints the output
+
+```{code-cell} ipython3
+x = 2 + 2
+print('x = ', x)
+```
+````
+</td>
+</tr>
+</table>
+
+The MyST-NB `.md` is much shorter, but it does not save the output `4`.
+
+
+## Pair your notebook files `.ipynb` and `.md`
+
+When you submit a Jupyter notebook to NumPy tutorials, we will convert
+it to a MyST-NB format. You can also submit the MyST-NB `.md` in your
+pull request.
+To keep the `.ipynb` and `.md` in sync--_or paired_--you need
+[Jupytext](https://jupytext.readthedocs.io/en/latest/index.html). 
+
+Install `jupytext` using:
+
+```
+pip install jupytext
+```
+or
+```
+conda install jupytext -c conda-forge
+```
+
+Once installed, start your `jupyter lab` or `jupyter notebook`
+session in the browser. When launching `jupyter lab` it will ask you to rebuild
+to include the Jupytext extension. 
+
+You can pair the two formats in the classic Jupyter, Jupyter Lab,
+or the command line:
+<ul>
+<details>
+    <summary>
+        <b>1. Classic Jupyter Jupytext pairing</b>.
+    </summary>
+    <img src="01-classic.gif" width=80% height=80%>
+</details>
+    
+<details>
+    <summary>
+        <b>2. Jupyter Lab Jupytext pairing</b>
+    </summary>
+    <img src="02-jupyterlab.gif" width=80% height=80%>
+</details>
+
+<details>
+    <summary>
+		<b>3. Command line Jupytext pairing</b>
+    </summary>
+  <pre><code>
+  jupytext --set-formats ipynb,myst notebook.ipynb
+  </pre></code>
+  Then, update either the MyST markdown or notebook file:
+  <pre><code>
+  jupytext --sync notebook.ipynb
+  </pre></code>
+</details>
+</ul>
+
+> __Note:__ With Jupytext installed, the classic Jupyter interface will
+> automatically open MyST files as notebooks. In JupyterLab, you can
+> right-click and choose "Open With -> Notebook" to open as a notebook.
+> The outputs of your code cells are only saved in the `.ipynb` file.
+
+## Wrapping up
+
+In this tutorial, you saw the json `.ipynb` and MyST-NB `.md` raw code 
+to create Jupyter notebooks. You can use both formats to create
+tutorials. Now you can work in either a simple text editor like VIM
+or emacs or continue building notebooks in your browser. Jupytext can
```

Comment:
Very nit: I think some people may object to your suggestion of vim or emacs as simple text editors :)

Reply:
haha! 
simple _text_ editor as opposed to a simple _text editor_. 

This was a conversation I first had in grad school: vim/emacs are editors for simple text i.e. ASCII characters. I would agree, they are not _simple_. 

Maybe cut out the _simple_?

Reply:
I think this is the last change before we merge this; I'd suggest something like *plain text editor*.